src/lib.rs

```diff
@@ -20,3 +20,9 @@ pub mod sorted_set {
     pub use momento_protos::cache_client::sorted_set_fetch_request::{Order, Range};
     pub use momento_protos::cache_client::SortedSetElement;
 }
+
+pub mod tokens {
```

Comment:
not sure if people have strong opinions on what this name should be

Reply:
maybe call it `token_expiry` and we can limit it to just this enum of values?  Just thinking we might wanna use token for something else in the future.  but this is also internal, 2-way door stuff that doesn't appear in any user-facing APIs, right?

Reply:
> maybe call it token_expiry

I like that

> we can limit it to just this enum of values?

Yep

>  but this is also internal, 2-way door stuff that doesn't appear in any user-facing APIs, right?

the `generate_api_token` method signature takes in an `Expiry` enum value

Reply:
in that case I think you need to create a custom enum directly in the SDK and map it.  You don't want to allow generated types (or generally, any types that are defined by one of your dependencies) leak through into your public API unless there's a really good reason for it.

Reply:
cc: @kvcache

Reply:
ah ok, I was trying to follow the pattern set by sorted sets, but that reasoning makes sense to me

Reply:
@kvcache lmk if you disagree on this but that's the general rule of thumb I have followed for this sort of thing over the years

Reply:
> unless there's a really good reason for it

I have _rarely_ found a good reason for it. They exist but usually only because of gaps in build tools as opposed to some inherent reason in a domain model...